src/client/components/Links/Link/Link.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Link as ReactRouterLink } from 'react-router-dom'
+
+import { LinkProps } from 'client/components/Links/Link/types'
+
+import { useLinkClassName } from './hooks/useLinkClassName'
+
+const Link: React.FC<LinkProps> = (props) => {
+  const { children, className, color, oneLine, rel, target, to } = props
+  // tooltip props
+  const { 'data-tooltip-content': tooltipContent, 'data-tooltip-id': tooltipId } = props
```

Comment:
i wouldnt do this, its clear from the name

one possibility also to pass 

`Props.tooltip = { id, content }`

Reply:
The component is just a wrapper . Using custom names , will require refactoring props if we switch component.
Wrapper components is suggested to have the primitive prop names . 
What do you think @sorja ?

Reply:
Even data col has the same prop names .
We should be consistent , this refactor shouldn't include how to consistently handle data tooltip props . If we really want to have different names , we should refactor all app , with even custom Div component to be consistent .

Reply:
Sure, my comment was more on

```
const {} = props
const {} = props
```